### ISSUE TITLE
Webpack dev server

### ISSUE THREAD

**Issue by MHadavand** [KitwareMedical/vtk.js-examples#3]
Trying to run webpack dev server using the same config from the repository and get this error

```Cannot find module 'webpack-cli/bin/config-yargs```

Changing the webpack version is a recommended solution but I get compatibility issues with kw-web-suite

**Comment by floryst** [comment]
Quickest solution is to run "npm install -D webpack-cli @webpack/serve", and then replace the script "webpack-dev-server" with "webpack serve" in package.json.

I'll update the examples for webpack 5 and latest kw-web-suite. Thanks for the report!

**Comment by MHadavand** [comment]
Thanks a lot! I had changed it to webpack serve and even installed webpack-cli but in order to get it to work, I had to delete node_moudles and the package-lock and reinstall dependencies. It is working now.

One other thing to add to the course material is the issue with passing variables from npm scripts on Windows. I had to install cross-env to get it work and use 

"cross-env EXAMPLE=cone.js webpack serve --mode=development"

Cheers,
